contracts/script/README.md

```diff
@@ -232,8 +232,8 @@ Solution: `export FOUNDRY_PROFILE=default`
 ## LightClient Contract Deployment
 
 ```bash
-forge script contracts/script/LightClient.s.sol:DeployLightClientContractScript $numBlocksPerEpoch $numInitValidators \
---sig 'run(uint32, uint32)' \
+forge script contracts/script/LightClient.s.sol:DeployLightClientContractScript $numBlocksPerEpoch $numInitValidators $maxHistorySeconds \
```

Comment:
We still use the old name `maxHistorySeconds` here.

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/1670/commits/33663d70d64e7f79785d992b90d9dfc97280bc79